lisa/microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -358,6 +396,18 @@ def run_metrics_tests(
         skip: Optional[List[str]] = None,
         subtest_timeout: Optional[int] = None,
     ) -> None:
+        # Initialize perf-stable profile (one-time per metrics test run)
+        # Metrics tests always use perf-stable setup
+        if not self._host_setup_done:
+            self._initialize_perf_stable_profile()
+            # One-time host setup (CPU, THP, irqbalance, storage, network)
+            self._setup_host_perf_policies()
+            self._setup_storage_hygiene()
+            self._setup_network_hygiene()
+            # One-time comprehensive warmup (storage + network + CPU/mem)
+            self._run_warmup_for_test("warmup")
```

Comment:
The argument is supposed to be the test case name right? If we're doing a one-time warmup we can get rid of the arg altogether.